Library/Homebrew/formula.rb

```diff
@@ -2608,6 +2609,19 @@ def dependencies_hash
     hash
   end
 
+  def dependencies_list(spec_symbol)
+    return if spec_symbol != :stable && spec_symbol != :head
+
+    send(spec_symbol)&.declared_deps&.each_with_object({}) do |dep, dep_hash|
+      next if dep.implicit? # Remove all implicit deps
+
+      dep_hash[dep.name] = {}.tap do |info|
```

Comment:
Don't feel strongly but unclear if `tap` helps or harms readability here 🤔

Reply:
Yeah, I tend to abuse `#tap` and `#then` a bit. I'll simplify it.